src/pages/documentation/mobile-core/api-reference.md

```diff
@@ -117,15 +117,16 @@ iOS
 
 <Tabs query="platform=react-native&api=log"/> --->
 
-## registerExtension(s)
+## registerExtensions
 
 Extensions are registered with Mobile Core so that they can dispatch and listen for events.
+This API can be used to register desired extensions and boot up the SDK for event processing. Calling `MobileCore.start()` API is no longer required when using `MobileCore.registerExtensions()`.
```

Comment:
We should also indicate that the MobileCore's `start` API is not required and deprecated starting v2.0.0

Reply:
Hmm wasn't aware the API was missing in the documentation. I'm fine adding it as part of #166, however when you are referring it here it still makes sense to call out that it's `no longer required and deprecated starting Mobile Core v2.0.0`. I'll leave it up to you to decide, I'm fine either ways.

Reply:
Done! I added it to the existing doc as advised.